examtool/examtool/cli/compile.py

```diff
@@ -71,6 +71,17 @@
     default=False,
     help="Generates a draft copy of the exam, which is faster but less accurate.",
 )
+@click.option(
+    "--allow-random-ids/--disallow-random-ids",
+    default=True,
+    help="Raises an error if an ID is not specified by a question in its config.",
+)
+@click.option(
+    "--threadcount",
+    default=16,
+    type=int,
+    help="The number of threads to process the json file (Default 16).",
```

Comment:
nit: JSON not json

Reply:
Also no need to put the default in the help

Reply:
Fixed